modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -391,7 +389,7 @@ private static string DeterminePlatformBclDir(string platform)
         /// </summary>
         private static bool PlatformRequiresCustomBcl(string platform)
         {
-            if (new[] { OS.Platforms.Android, OS.Platforms.iOS, OS.Platforms.HTML5 }.Contains(platform))
```

Comment:
VS Code's auto-format for C# does the opposite of this, adding spaces inside of braces. Is there a particular reason why they were removed? I think following rules set by common formatting tools is good.

Reply:
I think this goes according to our [style guide](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_style_guide.html#using-spaces).

>Do not use space:
>- ...
>- Within single line initializer braces.